data/serializer/src/main/java/tech/pegasys/teku/api/response/ValidatorStatusUtil.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Consensys Software Inc., 2025
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.api.response;
+
+import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+import tech.pegasys.teku.spec.datastructures.state.Validator;
+import tech.pegasys.teku.spec.datastructures.state.beaconstate.BeaconState;
+
+public class ValidatorStatusUtil {
```

Comment:
should we add unit test for this as there was for ValidatorResponse ?

Reply:
it was there, i definitely didnt want to lose it. i renamed it though so its going to be able to be found now.
Noticed EventTypeTest was in the wrong home now too, also fixed.